syne_tune/experiments/plotting.py

```diff
@@ -569,13 +575,19 @@ def _plot_figure(
             if not isinstance(legend_no, list):
                 legend_no = [legend_no]
             title_each_figure = subplots.title_each_figure
+            subplot_indices = (
+                list(range(len(stats)))
+                if subplots.subplot_indices is None
+                else subplots.subplot_indices
+            )
         else:
             subplot_xlims = None
             nrows = ncols = 1
             subplots_kwargs = dict(nrows=nrows, ncols=ncols)
             subplot_titles = None
             legend_no = [0]
             title_each_figure = False
+            subplot_indices = [0]
```

Comment:
Should this be set here?

Reply:
Yes, there is a loop over this list below.